### DISCUSSION TITLE
How to use Potplayer pbf import function ?

### DISCUSSION THREAD

**Discussion by fgt688** [mifi/lossless-cut#793] [Q&A]
Who can tell me how to use the import potplayer import bookmark function, in the import project to see this feature but do not know how to use, is to do special markers or how, thank you

怎么用导入potplayer书签？谁能告诉我怎么用导入potplayer 导入书签的功能，在导入项目里看到有这个功能但是不知道怎么用，是要做特别的标记还是怎样，谢谢，谷歌翻译不准勿怪囧

**Comment by mifi** [comment]
it will impoprt pbf markers as cut segments
